apps/teams-test-app/e2e-test-data/thirdPartyCloudStorage.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ThirdPartyCloudStorage",
+  "platforms": "Web",
+  "version": ">=2.18.0",
```

Comment:
I think this needs to be ">2.18.0" since this change isn't included in the latest 2.18.0 release

Reply:
Okay, modified it to >2.18.0